Wyam.Core/Modules/WriteFiles.cs

```diff
@@ -155,6 +155,7 @@ public virtual IEnumerable<IDocument> Execute(IReadOnlyList<IDocument> inputs, I
                             Metadata.Create(MetadataKeys.DestinationFileName, Path.GetFileName(path)),
                             Metadata.Create(MetadataKeys.DestinationFileDir, Path.GetDirectoryName(path)),
                             Metadata.Create(MetadataKeys.DestinationFilePath, path),
+                            Metadata.Create(MetadataKeys.RelativeDestinationFilePath, path.Substring(context.OutputFolder.Length)),
```

Comment:
Good idea - it makes total sense that you may need to know the relative path of a written file. ReadFiles already sets metadata for `RelativeFilePath`, `RelativeFilePathBase`, and `RelativeFileDir`. Any reason why we shouldn't just set the same metadata for consistency in WriteFiles?